launch/dynamo-run/src/subprocess/trtllm_inc.py

```
@@ -34,13 +44,63 @@
 # Default buffer size for kv cache events.
 DEFAULT_KV_EVENT_BUFFER_MAX_SIZE = 1024
 
 logging.basicConfig(level=logging.DEBUG)
 
 
+def parse_endpoint(endpoint: str) -> tuple[str, str, str]:
+    endpoint_str = endpoint.replace("dyn://", "", 1)
+    endpoint_parts = endpoint_str.split(".")
+    if len(endpoint_parts) != 3:
+        logging.error(
+            f"Invalid endpoint format: '{endpoint}'. Expected 'dyn://namespace.component.endpoint' or 'namespace.component.endpoint'."
```

Pylint (3.3.7)

[convention] Line too long (134/100)

(C0301)